pkg/daemon/kernelargs.go

```diff
@@ -50,33 +49,31 @@ func isArgTunable(arg string) (bool, error) {
 	return false, nil
 }
 
-// isArgInUse checks to see if the argument is already in use by the system currently
-func isArgInUse(arg, cmdLinePath string) (bool, error) {
-	if cmdLinePath == "" {
-		cmdLinePath = CmdLineFile
-	}
-	content, err := ioutil.ReadFile(cmdLinePath)
+// isKernelArgInUse checks to see if the argument is already in use by the system currently
+// On rpm-ostree systems, rpm-ostree owns the BLS configuration.
+func isKernelArgInUse(arg string) (bool, error) {
+	nu := NewNodeUpdaterClient()
+	checkable, err := nu.GetKernelArgs()
 	if err != nil {
 		return false, err
 	}
 
-	checkable := string(content)
-	if strings.Contains(checkable, arg) {
-		return true, nil
+	for _, v := range checkable {
+		if strings.Contains(v, arg) {
```

Comment:
/hold

I should fix this -- `strings.Contains` is wrong, it should be `strings.HasPrefix`. Otherwise `meh=baz` would match keys `meh` and `baz`.